### PR TITLE
Anki's to-do app

### PR DESCRIPTION
**Pull request by qwer-ctrl**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
-      Find me in src/app.js!
-    </div>
+    <Provider store={store}>
+      <BrowserRouter>
+      <Header />
+      <Navbar />
+      <Routes>
+        <Route path='/' element={<NotDoneTasks />}/>
+        <Route path='/navbar' element={<Navbar />} />
+        {/* <Route path='/notdonetasks' element={<NotDoneTasks />} /> */}
```
Comment (LovisaBrorson) [inline code/src/App.js:32]: Do you want to save row 32 for something?

File: code/src/reducers/tasks.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import { createSlice } from "@reduxjs/toolkit"
+import uniqid from 'uniqid'
+
+const tasks = createSlice({
+    name: 'tasks',
+    initialState: {
+        items: [
+            // {
```
Comment (LovisaBrorson) [inline code/src/reducers/tasks.js:8]: Is row 8-18 something you want to save?

**Review by LovisaBrorson — COMMENTED** [review]
Great to see your code! 
Very impressive, you have done a lot, I like that you added a header and a nav-bar. 
The code is easy and clean. 
You can mark the To-do list as done! ;)

File: code/src/components/DoneTasks.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+    const doneSummary = doneTasks.length
+
+
+     if (doneTasks < 1) {
+         return (
+            <section className="section-wrapper">
+            <img className="tasklist-img" src={state} alt="tasklist" />
+            </section>
```
Comment (sk-phan) [inline code/src/components/DoneTasks.js:35]: Nice that you display the empty task when all tasks are done.

File: code/src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import moment from "moment";
+
+const Header = () => {
+    const date = new Date()
+
+return(
+    <header>
+     <h1>to-do list</h1>
+     <p>{moment(date).format('dddd')}</p>
```
Comment (sk-phan) [inline code/src/components/Header.js:10]: I was stuck on how to use moment 😅. It's nice to see how you applied it here.

File: code/src/components/NotDoneTasks.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    const notDoneSummary = notDoneTasks.length
+
+     if (notDoneTasks < 1) {
+         return (
+            <>
+            <section className="section-wrapper">
+            <img className="tasklist-img" src={state} alt="tasklist" />
+            <AddTask />
```
Comment (sk-phan) [inline code/src/components/NotDoneTasks.js:36]: Perhaps you want to create a component for Empty State, so we do not need to rewrite these line of code many times

File: code/src/reducers/tasks.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+            //     isCaught: false
+            // }
+        ],
+    },
+    reducers: {
+        addTask: (store, action) => {
+            const newTask = {
+                id: uniqid(),
+                text: action.payload,
+                isCaught: false,
```
Comment (sk-phan) [inline code/src/reducers/tasks.js:26]: Just a recommendation to keep the store clean, we can pass these data directly in the onSubmitTask function, when we dispatch the input to the store. 😄

File: code/src/reducers/tasks.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+                }
+            })
+            store.items = updatedTaskItems
+        },
+
+        allTasksDone: (store) => {
+            const allDone = store.items.map((task) => {
+                return {
+                    ...task,
+                    isCaught: true
```
Comment (sk-phan) [inline code/src/reducers/tasks.js:57]: Great to see that you use the spread operator to update data

**Review by sk-phan — COMMENTED** [review]
Hi Anki, 

Great job! I just thought your app is so cute. I have shared minor ideas, hopefully, you find them helpful. 

Suki❤️

**Pull request closed without merging**